drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -143,38 +146,30 @@ internal fun checkComponentAvailability(
 
         val type = paymentMethod.type ?: throw CheckoutException("PaymentMethod type is null")
 
-        val provider = getProviderForType(type)
-        val configuration = dropInConfiguration.getConfigurationForPaymentMethod<Configuration>(type, application)
+        val availabilityCheck = getComponentAvailabilityCheckForType(type)
+        if (availabilityCheck == null) {
```

Comment:
I know this might be stupid but, I feel like using null here is not very semantic. Maybe we should have a `DefaultAvailabilityCheck` to return by  default instead?

Reply:
Hmmm good idea, but what would this class be? We can't make it an interface with a default value 🤔 
And what about PIX? We have to make a provider for it also if we want to return an `ComponentAvailabilityCheck` for every supported payment method type, right?
My idea was that we only provide a `ComponentAvailabilityCheck` for payment methods that might be not always available.
What do you think?